src/discussions/comments/comment/CommentHeader.jsx

```diff
@@ -47,15 +65,21 @@ function CommentHeader({
         />
       </div>
       <div className="d-flex align-items-center">
-        {comment.endorsed && (
-          <span className="btn-icon btn-icon-sm mr-1 align-items-center">
-            {
-              postType === 'question'
-                ? <Icon src={CheckCircle} className="text-success" data-testid="check-icon" />
-                : <Icon src={Verified} className="text-dark-500" data-testid="verified-icon" />
-            }
-          </span>
-        )}
+        <span className="btn-icon btn-icon-sm mr-1 align-items-center">
+          {actionIcons && (
+            <Icon
+              data-testid="check-icon"
+              onClick={
+                () => {
+                  handleIcons(actionIcons.action);
+                }
+              }
+              src={actionIcons.icon}
+              className={(actionIcons.id === 'endorse' || actionIcons.id === 'unendorse') ? 'text-dark-500' : 'text-success'}
```

Comment:
It can be change with this. Please confirm the colors with figma
['endorse', 'unendorse'].includes.(actionIcons.id))